entity/events.go

```diff
@@ -0,0 +1,35 @@
+package entity
+
+import "github.com/maas/gomaasclient/entity/event"
+
+// Event represent an event for MAAS node
+type Event struct {
+	UserName    string `json:"username,omitempty"`
+	Node        string `json:"node,omitempty"`
+	Hostname    string `json:"hostname,omitempty"`
+	Created     string `json:"created,omitempty"`
+	Type        string `json:"type,omitempty"`
+	Description string `json:"description,omitempty"`
+	Level       string `json:"level,omitempty"`
+	ID          int    `json:"id,omitempty"`
+}
+
+// EventsResp represents the MAAS Events endpoint
+type EventsResp struct {
+	Events []Event `json:"events,omitempty"`
+	Count  int     `json:"count,omitempty"`
```

Comment:
You are missing the pagination fields here:

```suggestion
	Count  int     `json:"count,omitempty"`
	NextURI string  `json:"next_uri,omitempty"`
	PrevURI string  `json:"prev_uri,omitempty"`
```

Reply:
I will try not to miss such an issue. Thanks @skatsaounis  :-)